app/models/derived_variables/case_log_variables.rb

```diff
@@ -83,7 +75,6 @@ def set_derived_fields!
       if location
         self.la = location.county
         self.postcode_full = location.postcode
-        self.unittype_sh = TYPE_OF_UNIT_MAP[location.type_of_unit]
         self.builtype = form.questions.find { |x| x.id == "builtype" }.answer_options.find { |_key, value| value["value"] == location.type_of_building }.first
```

Comment:
Did we want to remove the `builtype` stuff also?

Reply:
Yes, different commit, sorry!